addons/cookoff/stringtable.xml

```diff
@@ -25,6 +25,12 @@
             <Japanese>誘爆の損傷処理と砲塔の爆発効果を変更します。</Japanese>
             <Russian>Изменяет обработку урона для возгорания и эффекта срыва башни</Russian>
         </Key>
+        <Key ID="STR_ACE_CookOff_disableForAI_name">
+            <English>Disable cook off for AI vehicles</English>
+        </Key>
+        <Key ID="STR_ACE_CookOff_disableForAI_tooltip">
+            <English>Cook off will only run on player crewed vehicles</English>
```

Comment:
But that's only the case if the checkbox is actually set. Also missing a period at the end of the sentence.